Headers/Foundation/NSRunLoop.h

```diff
@@ -34,12 +34,18 @@ extern "C" {
 
 @class NSTimer, NSDate, NSPort;
 
+typedef NSString* NSRunLoopMode;
```

Comment:
Maybe we should use this new type for all the cases where mode is used here (and in the long run all the places where we call a run loop).

Reply:
Okay, I will make this change.